packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -213,8 +273,8 @@ export default class Meeting extends StatelessSparkPlugin {
   setRemoteStreams(pc) {
     // Receiving remote media streams
     pc.ontrack = (event) => {
-      [this.mediaStream, this.localMediaStream] = event.streams;
-
+      [this.mediaStream] = event.streams;
+      document.getElementById('remotevideo').srcObject = this.mediaStream;
```

Comment:
Was this just for testing? I'd like to revert and keep the stream -> video elements in the app and not the sdk

Reply:
yeah lets discuss tomorrow on new PR